### PR TITLE
Line widget upgrade

### PR DESCRIPTION
**Pull request by ocrossi**

### CODE REVIEW

**Comment by ocrossi** [comment]
@finetjul

**Comment by finetjul** [comment]
Can you please split into 2 MR ? one for the Reslice Cursor and one for the line widget

**Comment by ocrossi** [comment]
there are 2 different MR, but this one is based on the resliceCursor MR, since i need to have both changes applied to the vtk,js version i'm using in OFSEP. Please refer to #1758 for more information

File: Sources/Widgets/Representations/ArrowHandleRepresentation/Constants.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import vtkViewFinderSource from 'vtk.js/Sources/Filters/Sources/ViewFinderSource';
+
+export const ShapeSource = {
+  ARROWHEAD: vtkArrow2DSource,
+  CONE: vtkConeSource,
+  SPHERE: vtkSphereSource,
+  CUBE: vtkCubeSource,
+  CIRCLE: vtkCircleSource,
+  VIEWFINDER: vtkViewFinderSource,
+};
```
Comment (finetjul) [inline Sources/Widgets/Representations/ArrowHandleRepresentation/Constants.js:15]: Sorry, I was not clear enough, I did not mean to create an enum for the shape source.
I meant to create an enum for the vtkArrow2DSource different shapes.

Comment (finetjul) [reply Sources/Widgets/Representations/ArrowHandleRepresentation/Constants.js:15]: Please remove this shape mapping. The includes do not make it a "constant".

File: Sources/Widgets/Representations/ArrowHandleRepresentation/index.js [hunk 2]
```diff
@@ -222,6 +217,7 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
    *  - a first rotation to be oriented along model.rotation
    *  - an optional second rotation to face the camera
    * @param {vec3} scale3 Scale of the glyph, rotated when glyph is rotated.
+   * @returns {vec3} scale3 the vector
```
Comment (finetjul) [inline Sources/Widgets/Representations/ArrowHandleRepresentation/index.js:220]: @returns does not work exactly the same as for @Kitware/paraviewweb 
`* @returns {vec3} The input scale vector oriented`

**Comment by finetjul** [comment]
Already merged

**Pull request closed without merging**